packages/doctype-caip10-link/src/caip10-link-doctype.ts

```diff
@@ -67,7 +67,7 @@ export class Caip10LinkDoctype extends Doctype {
             throw new Error('Exactly one controller must be specified')
         }
 
-        const [address, linkedChainId] = metadata.controllers[0].split('@') // eslint-disable-line @typescript-eslint/no-unused-vars
+        const [, linkedChainId] = metadata.controllers[0].split('@') // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Can we remove the eslint suppression comment now?

Reply:
k